paf-mvp-pattern-library-audit/stories/components/AdvertImage.js

```diff
@@ -0,0 +1,57 @@
+import Details from './Details';
+import { BlockedImage } from './Icons';
+
+const Image = (args = {}) => {
```

Comment:
Why not using Typescript? (https://storybook.js.org/docs/react/configure/typescript)